backend/app/views/backoffice/accounts/_content_language_form.html.erb

```diff
@@ -1,5 +1,8 @@
 <%= simple_form_for [:backoffice, form_object] do |f| %>
   <%= f.error_notification %>
+  <% f.object.errors.each do |error| %>
```

Comment:
![image](https://user-images.githubusercontent.com/1286444/180821784-30d0cbde-ee69-4cdc-8e1c-a5b93b786dfa.png)
Maybe we need to refactor how errors look in backoffice, but not in this PR.

Reply:
:) yes, these error messages could definitely look better